chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/impl/CertificationResultDAOImpl.java

```diff
@@ -239,6 +253,29 @@ private CertificationResultEntity getEntityById(Long id) throws EntityRetrievalE
         List<CertificationResultEntity> result = query.getResultList();
         return result;
     }
+    
+    @Override
+    public List<CertificationResultDTO> findByCertifiedProductIdSED(Long certifiedProductId) {
+        List<CertificationResultEntity> entities = getEntitiesByCertifiedProductIdSED(certifiedProductId);
```

Comment:
This method doesn't appear to be called anywhere (at least not anywhere showing up in this compare view between branches. Delete?